plugins/slick.rowdetailview.js

```diff
@@ -156,13 +195,28 @@
         collapseItem(_expandedRows[i]);
       }
     }
-
+	
+    // Saves the current state of the detail veiw
+    function saveDetailView(item)
+    {
+        var view = $("#innerDeatilView_" + item.id);
```

Comment:
I assume there's a typo here and this `innerDeatilView_` should be `innerDetailView_`
Also `veiw` should be `view` in the comment and everyone it's used
You probably need to update at every place this variable is used, like couple lines below